modules/gojira-core/src/main/java/com/flipkart/gojira/core/ProfileRepository.java

```diff
@@ -81,6 +95,10 @@ public static void clearGlobalPerRequestID() {
     GLOBAL_PER_REQUEST_ID.remove();
   }
 
+  public static void clearThreadLocalProfileSetting() {
```

Comment:
method name can be renamed to requestProfileSetting.

Reply:
Done